src/hid-nintendo.c

```diff
@@ -909,7 +1178,7 @@ static void joycon_parse_report(struct joycon_ctlr *ctlr,
 	unsigned long msecs = jiffies_to_msecs(jiffies);
 
 	spin_lock_irqsave(&ctlr->lock, flags);
-	if (rep->vibrator_report &&
+	if (IS_ENABLED(CONFIG_NINTENDO_FF) && rep->vibrator_report &&
```

Comment:
The checks for CONFIG_NINTENDO_FF need to be removed or CONFIG_NINTENDO_FF needs to be hardcoded to true. Otherwise rumble won't work in the dkms module.